index.html

```diff
@@ -14,41 +15,139 @@
-      </p>
-    </article>
-    <section class="contact">
-      <p>LET’S CONNECT</p>
+    <section class="cards">
+
+      <div class="card">
+
+        <img class="cardimg" src="./images/tonic.png" alt="">
+
+        <div class="grid-item-title-1">
+          <p class="title">Tonic</p>
+          <div class="canopy">
+            <p>CANOPY</p>
+            <ul>
+              <li> Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <article>A daily selection of privately personalized reads;
+            no accounts or sign-ups required.</article>
+        </div>
+
+        <section class="button">
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>  
+          </ul>
+          <div class="proyect">
+            <a href="#"><button>See Proyect</button></a>
+          </div>
+        </section>
+
+      </div>
+      <div class="multi"></div>
+
+      
+
+      <div class="tonic2"></div>
+      <div class="multi2"></div>
     </section>
-    <section class="socialmedia">
-      <ul>
-        <li><i class="fa-brands fa-twitter"></i></li>
-        <li><i class="fa-brands fa-linkedin"></i></li>
-        <li><i class="fa-solid fa-m"></i></li>
-        <li><i class="fa-brands fa-square-github"></i></li>
-        <li><i class="fa-regular fa-hand-peace"></i></li>
-      </ul>
+    <section class="aboutmyself">
+      <h2>About Myself</h2>
+
+      <article class="job">
+        <p>
+          Hello I’m a software developer! I can 
+          help you build a product , feature or 
+          website. Look through some of my 
+          work and experience! If you like what 
+          you see and have a project you need 
+          coded, don’t hesitate to contact me.
+        </p>
+      </article>
+      <section class="contact">
+        <p>LET’S CONNECT</p>
+      </section>
+      <div class="get">
+        <a href="#"><button>Get my resume</button></a>
+      </div>
+
+      <section class="socialmedia">
+        <ul>
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin"></i></li>
+          <li><i class="fa-solid fa-m"></i></li>
+          <li><i class="fa-brands fa-square-github"></i></li>
+          <li><i class="fa-regular fa-hand-peace"></i></li>
+        </ul>
+      </section>
+
+      <section class="lenguage">
+        <h2>
+          Lenguage
+        </h2>
+      
+        <div class="logo">
+          
+            <div class="image"><img src="./images/Ellipse 1.png" alt=""></div>
+            <div class="leng"><p>Javascript</p></div>
+          
+        
+        
+          <div class="image"><img src="./images/Ellipse 1-1.png" alt=""></div>
+          <div class="leng"><p>HTML</p></div>
+        
+       
+          <div class="image"><img src="./images/Ellipse 1-2.png" alt=""></div>
+          <div class="leng"><p>CSS</p></div>
+        </div>
```

Comment:
- [ ] Good job adding some skills in your work , however it could be better if the design you get from your styling of this part sticks to the Figma design 👍 

| Your design | Figma design |
| :---: | :---: |
| ![Capture d’écran 2023-03-29 à 2 51 19 AM](https://user-images.githubusercontent.com/80458199/228402208-b9b4847e-4b0c-4495-8564-35d082aef6b1.png) | ![Capture d’écran 2023-03-29 à 3 23 50 AM](https://user-images.githubusercontent.com/80458199/228402431-77ef5cc5-78f8-4952-ab4c-6ca2271081ff.png) |

Reply:
@iambenkis  Thanks for the review I did take my time and fix all of them and more, Did learn a lot of UX design on this work, I know I'm no expert but I'll get there by time and practices.